clang/test/Driver/sycl-device-lib-amdgcn.cpp

```diff
@@ -42,3 +42,10 @@
 // CHK-ALL: [[DEVLIB:[0-9]+]]: input, "{{.*}}devicelib--amd.bc", ir, (device-sycl, gfx906)
 // CHK-ALL: {{[0-9]+}}: linker, {{{.*}}[[DEVLIB]]{{.*}}}, ir, (device-sycl, gfx906)
 
+// Check that llvm-link uses the "-only-needed" flag.
+// Not using the flag breaks kernel bundles.
+// RUN: %clangxx -### -std=c++11 -nogpulib --sysroot=%S/Inputs/SYCL \
```

Comment:
Is `-std=c++11` really needed for this test?

Reply:
Good point. I removed it